com.unity.multiplayer.mlapi/Runtime/Core/BatchingUtil.cs

```diff
@@ -0,0 +1,187 @@
+using MLAPI.Serialization.Pooled;
+using MLAPI.Serialization;
+using MLAPI.Configuration;
+using MLAPI.Profiling;
+using System.Collections.Generic;
+using System.Linq;
+using System;
+using System.IO;
+
+namespace MLAPI
+{
+    class BatchUtil
```

Comment:
Naming: doesn't really seem a 'Util', more of a BatchManager or MessageBatcher

Reply:
+1 for MessageBatcher and I also think it needs to be `internal`